vendor/README.md

```diff
@@ -7,6 +7,7 @@ This is a list of grammars that Linguist selects to provide syntax highlighting
 
```

Comment:
All the changes you've made to this file should be regenerated by running `script/list-grammars`. Make sure to do so after renaming the language entry.